packages/gatsby-source-wordpress/src/steps/set-gatsby-api-to-state.ts

```diff
@@ -38,7 +39,11 @@ const setGatsbyApiToState = (
       if (previewOptimizationPreset) {
         helpers.reporter.info(
           formatLogMessage(
-            `\nSince the "Preview Optimization" plugin option preset is enabled\nwe aren't fetching more than ${previewOptimizationPreset.options.type.__all.limit} nodes of each type.\nAdditionally, Gatsby image and static file links in HTML fields are disabled.\nIf you want to change this, please check the Preview docs for this plugin.\nhttps://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-source-wordpress/docs/features/preview.md`
+            `\nSince the "Preview Optimization" plugin option preset is enabled\n${
+              !usingGatsbyV4OrGreater
```

Comment:
Does this mean that preview optimization no longer works on V4 for the time being?

Reply:
It does unfortunately :( which means upgrading to v4 will break some user's previews. @jacksellwood we'll need to work on restoring this behaviour for WP & Gatsby v4 soon